code/centauri/hyperspace/parachain/src/parachain.rs

```diff
@@ -1 +1,55 @@
 include!(concat!(env!("OUT_DIR"), "/parachain.rs"));
+use api::runtime_types::dali_runtime::Call;
+use api::runtime_types::frame_system::extensions::check_nonce::CheckNonce;
+use codec::{Compact, Decode, Encode, Input};
+use sp_runtime::generic::Era;
+use subxt::Config;
+
+#[derive(Encode, Decode, Clone, Eq, PartialEq)]
+pub struct ChargeAssetTxPayment {
+	#[codec(compact)]
+	tip: u128,
+	asset_id: Option<u128>,
+}
+
+pub type SignedExtra = (Era, CheckNonce, ChargeAssetTxPayment);
+
+pub struct UncheckedExtrinsic<T: Config> {
+	pub signature: Option<(<T as Config>::Address, sp_runtime::MultiSignature, SignedExtra)>,
+	pub function: Call,
+}
```

Comment:
Why not use the one in `sp_runtime`

Reply:
@seunlanlege I tried this option at first, but it requires the `SignedExtension` trait to be implemented for `SignedExtra`. If it's fine to have a blank implementation of it, I can use it